www/tilesource-iiif.html

```diff
@@ -3,14 +3,14 @@
 </h2>
 <p>
     The latest version (1.1) of the International Image Interoperability Framework: Image API is formally described
```

Comment:
latest version (2.0) ... which will be out of date in a couple of months when we rev to 2.1. Might be worth not saying "latest" ?